lib/ex_grok/master_supervisor.ex

```diff
@@ -0,0 +1,17 @@
+defmodule ExGrok.MasterSupervisor do
+  @moduledoc false
+  use Supervisor
+
+  def start_link do
+    Supervisor.start_link(__MODULE__, [], name: :master_supervisor)
+  end
+
+  def init(_) do
+    children = [
+      worker(ExGrok.Ngrok, []),
```

Comment:
Killing this worker did not seem to stop/restart the Ngrok process on my machine, instead the application shut down.